curriculum/challenges/english/20-upcoming-python/learn-list-comprehension-by-building-a-case-converter-program/657f4add33ea4b1f61ba3dc8.md

```diff
@@ -0,0 +1,46 @@
+---
+id: 657f4add33ea4b1f61ba3dc8
+title: Step 22
+challengeType: 20
+dashedName: step-22
+---
+
```

Comment:
The code is already running. I have no idea for this step. Maybe ask to pass a different input?

Reply:
Just did that.